### PR TITLE
New package: aoguai.QQtoExcel version 1.0

### PR DESCRIPTION
**Pull request by DuckDuckStudio**
### 拉取请求使用此工具创建： [komac](https://github.com/russellbanks/Komac) v2.9.0 :rocket:

- https://github.com/microsoft/winget-pkgs/pull/201276#issuecomment-2558327640

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/201276)

### CODE REVIEW

File: manifests/a/aoguai/QQtoExcel/1.0/aoguai.QQtoExcel.locale.zh-CN.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+- 备份
+- 聊天记录备份
+- 分析
+- 统计
+- 可视化
+- 数据
+- 用户画像
+- NLP
+- 整理
+- qq
```
Comment (DuckDuckStudio) [inline manifests/a/aoguai/QQtoExcel/1.0/aoguai.QQtoExcel.locale.zh-CN.yaml:48]:
```suggestion
- qq
ReleaseNotes: |
  V1.0.0-20220719
  初版
    支持好友/群聊/全部聊天记录 转换导出
    支持可选项 选择导出
```

File: manifests/a/aoguai/QQtoExcel/1.0/aoguai.QQtoExcel.locale.zh-CN.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+ReleaseNotes: |
+  V1.0.0-20220719
+  初版
+    支持好友/群聊/全部聊天记录 转换导出
+    支持可选项 选择导出
+ReleaseNotesUrl: https://github.com/aoguai/QQtoExcel/releases/tag/V1.0.0-20220719
```
Comment (DuckDuckStudio) [inline manifests/a/aoguai/QQtoExcel/1.0/aoguai.QQtoExcel.locale.zh-CN.yaml:54]:
```suggestion
ReleaseNotesUrl: https://github.com/aoguai/QQtoExcel/releases/tag/V1.0.0-20220719
Documentations: 
- DocumentLabel: Wiki
  DocumentUrl: https://github.com/aoguai/QQtoExcel/wiki
```

**Comment by DuckDuckStudio** [comment]
image
image

**Comment by DuckDuckStudio** [comment]
Retry-1

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> 2024-12-24 17:55:43.965 [FAIL] Installer failed security check. Url: https://github.com/aoguai/QQtoExcel/releases/download/V1.0.0-20220719/QQtoExcelV1.0_windows.zip Result: 0x80004005

(Automated response - build 931.)

**Comment by DuckDuckStudio** [comment]
> Manual Validation ended with:
> 
> > 2024-12-24 17:55:43.965 [FAIL] Installer failed security check. Url: https://github.com/aoguai/QQtoExcel/releases/download/V1.0.0-20220719/QQtoExcelV1.0_windows.zip Result: 0x80004005
> 
> (Automated response - build 931.)

https://www.virustotal.com/gui/file/57d9278804693de3205a0549a45dba64c5e682d1134da86839736b274885648e/detection
image

**Comment by stephengillie** [comment]
Sorry, the virus signature must have been old.

Verify:
image
The VM isn't prepared to display the characters.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**